Source/Core/Core/PowerPC/JitArm64/JitArm64_Branch.cpp

```diff
@@ -168,7 +177,7 @@ void JitArm64::bcx(UGeckoInstruction inst)
   }
   else
   {
-    WriteExit(js.op->branchTo, inst.LK, js.compilerPC + 4);
+    WriteExit(js.op->branchTo, inst.LK, js.compilerPC + 4, WA);
```

Comment:
This is unsafe because `WA` is being passed as a scratch register to `gpr.Flush`.